ckanext/xloader/tests/samples/sample_with_empty_lines.csv

```diff
@@ -0,0 +1,8 @@
+date,temperature,place
+2011-01-01,1,Galway
+2011-01-02,-1,Galway
+2011-01-03,0,Galway
+2011-01-01,6,Berkeley
+,,Berkeley
+2011-01-03,5,
+
```

Comment:
please test with 2 new lines as single new line is standard.

Reply:
Added.